_data/sites.json

```diff
@@ -6003,6 +6003,17 @@
         ]
     },
 
+    {
+        "name": "RapidAPI",
+        "url": "https://rapidapi.com/contact-us",
+        "difficulty": "hard",
+        "notes": "Account can be only deleted with sending request to support",
+        "email": "support@rapidapi.com",
+        "domains": [
+            "rapidapi.com"
+        ]
+    },
+
```

Comment:
This needs to be moved upwards to keep the alphanumeric ordering and pass on the Travis CI
Put it between "Rainforest QA" and "RateYourMusic"